daemon/network.go

```diff
@@ -1005,32 +1005,32 @@ func getPortMapInfo(sb *libnetwork.Sandbox) nat.PortMap {
 	}
 
 	for _, ep := range sb.Endpoints() {
-		pm, _ = getEndpointPortMapInfo(ep)
```

Comment:
Instead of ignoring the error, maybe we should return or at least log it?

Reply:
https://github.com/moby/moby/blob/4d62124a7963eba825f540ce6b40ae93314daac6/daemon/network.go#L1024 is this something that could actually happen or it's just a theoretical error?

Reply:
Either way, we could at least log it here or after the `nat.NewPort`

Reply:
Done